src/components/Form/Form.test.tsx

```diff
@@ -106,6 +110,20 @@ const ComplexForm = ({
         validate={validateDate}
       />
       <Field label="Test Frequency" name="testFrequency" as={Frequency} />
+      <Field
+        label="Test Checkbox"
+        name="testCheckbox"
+        testId="field"
```

Comment:
same test id as the `Textarea` below.

Reply:
yes, it's so that they can all be targeted at the same time to test for invalid when submitting, instead of testing each of them individually.